packages/bruno-app/src/utils/codegenerator/auth.js

```
@@ -11,8 +11,8 @@ export const getAuthHeaders = (requestAuth, collection = null, item = null) => {
 
   switch (requestAuth.mode) {
     case 'basic':
-      const username = get(requestAuth, 'basic.username', '');
-      const password = get(requestAuth, 'basic.password', '');
+      const username = get(requestAuth, 'basic.username') || '';
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)